sidecar-injector/pkg/inject/common.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/golang/glog"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"fmt"
```

Comment:
Imports should probably be sorted by name and split by built-ins vs external packages

Reply:
done